cytominer_eval/evaluate.py

```diff
@@ -100,7 +101,7 @@ def evaluate(
         Only used when `operation='mp_value'`. A key, item pair of optional parameters
         for calculating mp value. See also
         :py:func:`cytominer_eval.operations.util.default_mp_value_parameters`
-    percentile : float, optional
+    percentile : float or list of floats, optional
```

Comment:
```suggestion
    enrichment_percentile : float or list of floats, optional
```

To be consistent with the function arg